api/pyproject.toml

```diff
@@ -26,3 +38,133 @@ multi_line_output = 3
 sections = "FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,TESTS,LOCALFOLDER"
 skip_gitignore = true
 use_parentheses = true
+
+[tool.mypy]
+python_version = "3.10"
```

Comment:
mypy semble attendre sa version en 3.10 et non "3.10" (en tout cas dans mypy.ini c'était la syntaxe)

Reply:
Dans le fichier INI, toutes les valeurs sont des chaîne de caractères (ou des listes de chaînes). Donc quand j'ai écrit `python_version = "3.10"`, c'était effectivement une erreur car mypy lisait ça comme la chaîne de caractères `"3.10"`, avec des guillemets.

Ici, en TOML, le type des variables est plus clair : `"3.10"` est une chaîne de caractères, `3.10` (sans les guillemets) est un nombre... que mypy ne comprend pas correctement :  

    $ mypy monfichier.py
    pyproject.toml: [mypy]: python_version: Python 3.1 is not supported (must be 3.4 or higher)

Fournit la valeur sous la forme d'une chaîne de caractères est donc à mon avis la bonne solution en TOML.